docs/dev/cypress.rst

```diff
@@ -18,6 +18,206 @@ the external services (Jira, Bitbucket and Bamboo) and connect them directly wit
 Therefore, the current setup only dynamically deploys the Artemis server and configures it to connect to
 the prelive system, which is already properly setup in the university data center.
 
+
+Local Cypress Setup
+-------------------
+Sometimes developers need to setup Cypress locally, in order to debug failing E2E tests or write new tests.
+In order to setup cypress locally, we need to follow these steps:
+
+1. Install dependencies
+
+  First head into the cypress folder by using ``cd /src/test/cypress``. Now run ``npm install``.
+
+2. Customize Cypress settings
+
+  To connect cypress to our local Artemis instance, we need to adjust some configurations. 
+  First we need to set the URL or IP of the Artemis instance in the ``cypress.config.ts`` file.
+  Adjust the ``baseUrl`` setting to fit your setup (e.g. ``baseUrl: 'http://localhost:9000',``)
+
+3. Adjust user settings
+
+  We also need to adjust the user setting, which will determine the usernames and passwords, that cypress
+  will use. These settings are located within the ``cypress.env.json`` file. If you use the Atlassian setup,
+  the file should typically look like this:
+   
+  .. code-block:: json
+
+    {
+      "username": "artemis_test_user_USERID",
+      "password": "artemis_test_user_USERID",
+      "adminUsername": "artemis_admin",
+      "adminPassword": "artemis_admin",
+      "allowGroupCustomization": true,
+      "studentGroupName": "students",
+      "tutorGroupName": "tutors",
+      "editorGroupName": "editors",
+      "instructorGroupName": "instructors"
+    }
+
+  The ``USERID`` part will be replaced by different user ids. These are set within the ``support/users.ts`` file. 
```

Comment:
To strengthen that cypress will do this and you don't need to change that manually in the file:
```suggestion
  The ``USERID`` part will be automatically replaced by different user ids. These are set within the ``support/users.ts`` file. 
```